2] loto/main.py

```diff
@@ -0,0 +1,75 @@
+from random import randint
+
+
+def nums_of_player() -> list:
+    """
+    The function gets a list of six original numbers
+    of the player.
+
+    Funktion checks whether:
+        player entered integer in the range 1-49,
+        player has not entered a given number before.
+
+    Funktion return a sorted list of numbers.
+    """
+    # gettint a list of numbers
+    player_nums = []
```

Comment:
lepsi je set protoze delas memebership test s pouzitim in operatoru. Set je na tuhle operaci lepe staveny, ma konstatni casovou komplexitu pro ziskani prvku, zatimco list linearni - ty listy jsou tady kratke ale i tak je set vzdy lepsi

Reply:
Opraveno